BackendNew/models/students.js

```diff
@@ -602,9 +602,26 @@ exports.getStudentById = (studentId, sendStudent) => {
                               companiesMap.get(value.batchId).push(value.Name);
                             });
                             for (let batch of batchesMap.keys()) {
-                              batchesMap
-                                .get(batch)
-                                .push({ companies: companiesMap.get(batch) });
+                              var val = batchesMap.get(batch);
+                              var cv;
+                              var stratDate;
+                              var selectedCompany;
+                              val.forEach(value => {
+                                if (value.cv) {
+                                  cv = value.cv;
+                                } else if (value.dateOfStart) {
+                                  stratDate = value.dateOfStart;
+                                } else if (value.selectedCompany) {
+                                  selectedCompany = value.selectedCompany;
+                                }
+                              });
+                              var modifiedSet = {
+                                cv: cv,
+                                stratDate: stratDate,
```

Comment:
startDate

Reply:
@hasinichathu What do you mean by startdate?

Reply:
If you asked if there is no start date. This is the sample query for that.

```
{
    "studentdata": {
        "IndexNumber": "164120F",
        "FullName": "Chamodi Sandeepani",
        "NameWithInitials": "SANDEEPANI K.G.C.",
        "PhoneNumber": 716931384,
        "Email": "chamodisandeepani@gmail.com",
        "Sem1GPA": 3.89,
        "Sem2GPA": 3.78,
        "Sem3GPA": 4.03,
        "Sem4GPA": 3.57,
        "SGPA": 3.72,
        "PreferedArea1": "UI/UX",
        "PreferedArea2": "Networking",
        "PreferedArea3": "QA Engineering"
    },
    "batchInfo": {
        "batch16": {
            "cv": "cv",
            "companies": []
        }
    }
}
```

Reply:
spellings should be startDate